packages/eslint-plugin/docs/rules/member-ordering.md

```diff
@@ -1290,6 +1312,21 @@ The third grouping option is to ignore both scope and accessibility.
 ]
 ```
 
+### Member Group Types (Readonly Fields)
+
+It is possible to group fields by their `readonly` modifiers.
+
+```jsonc
+[
+  // Index signature
+  // No grouping for index signature.
```

Comment:
Index signatures can be `readonly`, though. And you can even have a `: number` & `: string` in the same type:

```ts
interface Example {
    [i: number]: number;
    readonly [i: string]: number;
}
```

Cases of having one or both of those be mutable or not should be in the unit tests.

Reply:
I actually miss this one. I didn't realize that index signatures could be readonly too. I will try to update the code and add more test cases for this.